src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,43 @@
+.clock {
+  @include clock-shape($clock-main-size);
+
+  position: relative;
+  background-color: $clock-main-color;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+
+  &__center-elem {
+    @include clock-shape($clock-center-elem-size);
+
+    position: absolute;
+    z-index: 2;
+    top: ($clock-main-size / 2) - ($clock-center-elem-size / 2);
+    background-color: $clock-center-color;
+  }
+
+  &__minutes {
+    position: absolute;
+    top: ($clock-main-size / 2) - $arrow-minutes-height;
+    width: $arrow-minutes-width;
+    height: $arrow-minutes-height;
```

Comment:
Mixin square

Reply:
@Ihor-Karpyn but, that is not a squre, these variables have different value for width and height. Shoud I rewrite that with mixin?